libs/nx-remotecache-s3/src/lib/s3-client.ts

```diff
@@ -6,10 +6,10 @@ import { getDefaultRoleAssumerWithWebIdentity } from '@aws-sdk/client-sts';
 import type { S3ClientConfig } from '@aws-sdk/client-s3/dist-types/S3Client';
 import type { DefaultProviderInit } from '@aws-sdk/credential-provider-node/dist-types/defaultProvider';
 
-const ENV_ENDPOINT = 'NX_CACHE_S3_ENDPOINT';
-const ENV_PROFILE = 'NX_CACHE_S3_PROFILE';
-const ENV_FORCE_PATH_STYLE = 'NX_CACHE_S3_FORCE_PATH_STYLE';
-const ENV_REGION = 'NX_CACHE_S3_REGION';
+const ENV_ENDPOINT = 'NXCACHE_S3_ENDPOINT';
```

Comment:
I'll merge this one last, as a separate PR, since it'll trigger a major version update

Reply:
Ok I can split this out into a separate PR if you would like

Reply:
Sure, that would help :-)